### PR TITLE
Blacklist some gpu

### PR DESCRIPTION
**Pull request by sum2012**
Not ready because offical version v1.0.1-880-g1fd6cf1 black screen in my phone in Tactics Ogre: Let Us Cling Together

### CODE REVIEW

File: GPU/GLES/GLES_GPU.cpp [hunk 1]
```diff
@@ -56,6 +56,13 @@ enum {
 	FLAG_DIRTYONCHANGE = 64,
 };
 
+static const char *BlacklistGPUNames[] = {
```
Comment (hrydgard) [inline GPU/GLES/GLES_GPU.cpp]: Rename this "FramebufferFetchBlacklist" to be clear what feature it is blacklisting for.

Comment (sum2012) [inline GPU/GLES/GLES_GPU.cpp]: Done

**Comment by hrydgard** [comment]
Yeah, there are some new issues after the cleanup, I will debug them on my devices too.

**Comment by hrydgard** [comment]
@sum2012  I found the blackscreen, no need for you to bisect. Fix coming.

**Comment by sum2012** [comment]
@hrydgard Good news

**Comment by sum2012** [comment]
My phone fixed :)
@shinemu Try this build
(Test queen gate spiral chaos)
https://drive.google.com/file/d/0B3OaSdeV0L8keHVKdjZZbjN5NW8/view?usp=sharing

**Comment by unknownbrackets** [comment]
It's a shame not even PowerVR Rogue G6430 will work.  If those don't work it seems like we want to blacklist even more - any rationale for allowing "PowerVR 5" at all?  Though, I wonder if driver versions are a factor.

-[Unknown]

**Comment by hrydgard** [comment]
I wonder if we are using it wrong or something. It can't be that broken, can it?

**Comment by unknownbrackets** [comment]
Not impossible.  This is essentially what we do with it:

https://github.com/hrydgard/ppsspp/blob/1fd6cf1cc56c350ea0567b778d93a218c4ddd97e/GPU/GLES/FragmentShaderGenerator.cpp#L956

Basically, when the extension exists, we just read the last frag data rather than using an expensive-and-outdated-anyway texture copy of the render target.  I remember someone reported to me that a game was improved by this, but I never actually had a device that supported the feature and so couldn't do a ton of testing.  Could definitely be broken.

Theoretically it's something we want to do, though.

-[Unknown]

**Comment by sum2012** [comment]
Can we add a option instead ?

**Comment by shinemu** [comment]
@sum 2012 yes it works too sir ^_^ i'm curious,why playing lost heroes on
iphone 6 is laggy so bad while on android power vr is good,aren't they both
using power vr? Just curious sir nothing to concern about

**Comment by ThePhoenixRises** [comment]
@unknownbrackets 
This maybe what you mentioned about PowerVR 
https://github.com/hrydgard/ppsspp/pull/7182

**Pull request merged**